index.bs

```diff
@@ -155,8 +155,7 @@ explainer documents.
 <em>This section is non-normative</em>.
 
 The scope of this specification is currently limited
-to specifying primitives
-which enable expose data from local sensors.
+to specifying primitives which enable expose data from [=device sensors=].
```

Comment:
`enable user agent to expose data` or `enable exposure of the data`, maybe something else :smile:, `enable expose` sounds strange.

Reply:
s/expose/exposing/